packages/ui-extensions/src/surfaces/customer-account/api/docs.ts

```diff
@@ -117,6 +117,12 @@ export interface Docs_ResourceItem_Button_Action
     | 'kind'
   > {}
 
+export interface Docs_Menu_Button_Action
+  extends Omit<
+    ButtonProps,
+    'kind' | 'textDecoration' | 'inlineAlignment' | 'inlineSize' | 'size'
+  > {}
```

Comment:
I omitted props overriden in https://github.com/Shopify/checkout-web/blob/main/packages/checkout-web-ui/src/components/Button/withContextualStyle.tsx#L23